src/server.c

```diff
@@ -469,55 +473,55 @@ static bool handle_server_work(PgSocket *server, PktHdr *pkt)
 				return false;
 			}
 		}
-		pop_outstanding_request(server, "E", &ignore_packet);
+		pop_outstanding_request(server, (char[]) {PqMsg_Execute, '\0'}, &ignore_packet);
 
 		break;
 
-	case 'N':		/* NoticeResponse */
+	case PqMsg_NoticeResponse:
 		break;
 
 	/* reply to LISTEN, don't change connection state */
-	case 'A':		/* NotificationResponse */
+	case PqMsg_NotificationResponse:
 		idle_tx = server->idle_tx;
 		ready = server->ready;
 		async_response = true;
 		break;
 
 	/* copy mode */
-	case 'G':		/* CopyInResponse */
-	case 'W':		/* CopyBothResponse */
+	case PqMsg_CopyInResponse:
+	case PqMsg_CopyBothResponse:
 		slog_debug(server, "COPY started");
 		server->copy_mode = true;
 		break;
-	case 'H':		/* CopyOutResponse */
+	case PqMsg_CopyOutResponse:
 		break;
 	/* chat packets */
-	case '1':		/* ParseComplete */
-		pop_outstanding_request(server, "P", &ignore_packet);
+	case PqMsg_ParseComplete:
+		pop_outstanding_request(server, (char[]) {PqMsg_Parse, '\0'}, &ignore_packet);
 		break;
-	case '2':		/* BindComplete */
-		pop_outstanding_request(server, "B", &ignore_packet);
+	case PqMsg_BindComplete:
+		pop_outstanding_request(server, (char[]) {PqMsg_Bind, '\0'}, &ignore_packet);
 		break;
-	case '3':		/* CloseComplete */
-		pop_outstanding_request(server, "C", &ignore_packet);
+	case PqMsg_CloseComplete:
+		pop_outstanding_request(server, (char[]) {PqMsg_Close, '\0'}, &ignore_packet);
 		break;
-	case 'n':		/* NoData */
-	case 'T':		/* RowDescription */
-		pop_outstanding_request(server, "D", &ignore_packet);
+	case PqMsg_NoData:
+	case PqMsg_RowDescription:
+		pop_outstanding_request(server, (char[]) {PqMsg_Describe, '\0'}, &ignore_packet);
 		break;
-	case 't':		/* ParameterDescription */
-	case 'c':		/* CopyDone(F/B) */
-	case 'f':		/* CopyFail(F/B) */
-	case 'V':		/* FunctionCallResponse */
+	case PqMsg_ParameterDescription:
+	case PqMsg_CopyDone:
+	case PqMsg_CopyFail:
+	case PqMsg_FunctionCallResponse:
 		break;
-	case 'I':		/* EmptyQueryResponse == CommandComplete */
-	case 's':		/* PortalSuspended */
-		pop_outstanding_request(server, "E", &ignore_packet);
+	case PqMsg_EmptyQueryResponse:	/* EmptyQueryResponse == CommandComplete XXX */
```

Comment:
Agreed that this comment is confusing

```suggestion
	case PqMsg_EmptyQueryResponse:	/* EmptyQueryResponse is similar to CommandComplete, which is handled above */
```

Reply:
done